scripts/verify-build-artifacts.js

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import fs from 'fs';
+import path from 'path';
+import glob from 'fast-glob';
+import 'colors';
+
+function verifyCustomElementsJson() {
+    // Components that don't need their own custom-elements.json manifest
+    const customElementsIgnoreList = [
+        'packages/modal',
+        'packages/iconset',
+        'packages/clear-button',
+        'packages/close-button',
+    ];
+
+    const packages = glob.sync('packages/*/', { onlyDirectories: true });
+    for (const pkg of packages) {
+        // Remove trailing slash for comparison
+        const pkgPath = pkg.replace(/\/$/, '');
+        if (customElementsIgnoreList.includes(pkgPath)) {
+            continue;
+        }
+        const customElementsPath = path.join(pkg, 'custom-elements.json');
+        if (!fs.existsSync(customElementsPath)) {
+            throw new Error(`Missing custom-elements.json in ${pkg}`);
+        }
+    }
+}
+
+function verifyVersionJs() {
+    const basePackageJson = JSON.parse(
+        fs.readFileSync('tools/base/package.json', 'utf8')
```

Comment:
can we put this in a try catch block to handle is this is missing or unreadable?